src/fortranapex/magfld.f

```diff
@@ -1,6 +1,6 @@
 C  FILE NAME: magfld.f
 
-      SUBROUTINE COFRM (DATE)
+      SUBROUTINE COFRM (DATE, FILENAME)
```

Comment:
Because you added the FILENAME variable here, it needs to be declared in the variable block down below as a string of length 1000.  I'd make a suggestion, but github doesn't allow it.

Reply:
I'm not 100% sure what you mean. It is declared here:
https://github.com/aburrell/apexpy/blob/b28d0f446ef174a8f404c3f37082e19678e59a52/src/fortranapex/magfld.f#L121

Is that what you mean or somewhere else?